src/SharpRaven.sln

```diff
@@ -45,32 +37,59 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "docs", "docs", "{F44F1C47-D
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SharpRaven.Nancy.UnitTests", "tests\SharpRaven.Nancy.UnitTests\SharpRaven.Nancy.UnitTests.csproj", "{7FE52A2B-430C-4C6F-BEA9-0855AF973D0C}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SharpRaven.UnitTests", "tests\SharpRaven.UnitTests\SharpRaven.UnitTests.csproj", "{E1DBEBBF-9448-4D99-B378-2B8CF1629F31}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SharpRaven.WebTest", "tests\SharpRaven.WebTest\SharpRaven.WebTest.csproj", "{156621FC-2C48-4CDF-A368-9347BABE9089}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug 3.5|Any CPU = Debug 3.5|Any CPU
+		Debug 3.5|Mixed Platforms = Debug 3.5|Mixed Platforms
+		Debug 3.5|x86 = Debug 3.5|x86
 		Debug 4.0|Any CPU = Debug 4.0|Any CPU
 		Debug 4.0|Mixed Platforms = Debug 4.0|Mixed Platforms
 		Debug 4.0|x86 = Debug 4.0|x86
 		Debug 4.5|Any CPU = Debug 4.5|Any CPU
 		Debug 4.5|Mixed Platforms = Debug 4.5|Mixed Platforms
 		Debug 4.5|x86 = Debug 4.5|x86
+		Release 3.5|Any CPU = Release 3.5|Any CPU
+		Release 3.5|Mixed Platforms = Release 3.5|Mixed Platforms
+		Release 3.5|x86 = Release 3.5|x86
 		Release 4.0|Any CPU = Release 4.0|Any CPU
 		Release 4.0|Mixed Platforms = Release 4.0|Mixed Platforms
 		Release 4.0|x86 = Release 4.0|x86
 		Release 4.5|Any CPU = Release 4.5|Any CPU
 		Release 4.5|Mixed Platforms = Release 4.5|Mixed Platforms
 		Release 4.5|x86 = Release 4.5|x86
+		Release|Any CPU = Release|Any CPU
+		Release|Mixed Platforms = Release|Mixed Platforms
+		Release|x86 = Release|x86
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{CC80A2E1-AE39-44DE-8DA3-4EEF42F90FB1}.Debug 3.5|Any CPU.ActiveCfg = Debug 3.5|Any CPU
+		{CC80A2E1-AE39-44DE-8DA3-4EEF42F90FB1}.Debug 3.5|Any CPU.Build.0 = Debug 3.5|Any CPU
+		{CC80A2E1-AE39-44DE-8DA3-4EEF42F90FB1}.Debug 3.5|Mixed Platforms.ActiveCfg = Debug 3.5|Any CPU
+		{CC80A2E1-AE39-44DE-8DA3-4EEF42F90FB1}.Debug 3.5|Mixed Platforms.Build.0 = Debug 3.5|Any CPU
+		{CC80A2E1-AE39-44DE-8DA3-4EEF42F90FB1}.Debug 3.5|x86.ActiveCfg = Debug 3.5|Any CPU
 		{CC80A2E1-AE39-44DE-8DA3-4EEF42F90FB1}.Debug 4.0|Any CPU.ActiveCfg = Debug 4.0|Any CPU
 		{CC80A2E1-AE39-44DE-8DA3-4EEF42F90FB1}.Debug 4.0|Any CPU.Build.0 = Debug 4.0|Any CPU
 		{CC80A2E1-AE39-44DE-8DA3-4EEF42F90FB1}.Debug 4.0|Mixed Platforms.ActiveCfg = Debug 4.0|Any CPU
 		{CC80A2E1-AE39-44DE-8DA3-4EEF42F90FB1}.Debug 4.0|Mixed Platforms.Build.0 = Debug 4.0|Any CPU
 		{CC80A2E1-AE39-44DE-8DA3-4EEF42F90FB1}.Debug 4.0|x86.ActiveCfg = Debug 4.0|Any CPU
-		{CC80A2E1-AE39-44DE-8DA3-4EEF42F90FB1}.Debug 4.5|Any CPU.ActiveCfg = Debug 4.5|Any CPU
-		{CC80A2E1-AE39-44DE-8DA3-4EEF42F90FB1}.Debug 4.5|Any CPU.Build.0 = Debug 4.5|Any CPU
+		{CC80A2E1-AE39-44DE-8DA3-4EEF42F90FB1}.Debug 4.5|Any CPU.ActiveCfg = Debug 3.5|Any CPU
+		{CC80A2E1-AE39-44DE-8DA3-4EEF42F90FB1}.Debug 4.5|Any CPU.Build.0 = Debug 3.5|Any CPU
```

Comment:
These two build configurations look weird. `Debug 4.5` and `Debug 3.5` in the same configuration?